apps/studio/components/interfaces/Storage/VectorBuckets/CreateVectorTableSheet.tsx

```diff
@@ -86,7 +98,11 @@ const formId = 'create-vector-table-form'
 
 export type CreateVectorTableForm = z.infer<typeof FormSchema>
 
-export const CreateVectorIndexSheet = () => {
+interface CreateVectorTableSheetProps {
+  bucketName: string
```

Comment:
So we can reference the `bucketName` in the _Target schema_ field. Honestly I might do away with this field entirely, later. Not sure how helpful it is for people to know this.